app/src/main/java/com/zeapo/pwdstore/git/GitServerConfigActivity.kt

```diff
@@ -68,6 +71,27 @@ class GitServerConfigActivity : BaseGitActivity() {
         binding.serverUrl.setText(GitSettings.url)
         binding.serverBranch.setText(GitSettings.branch)
 
+        binding.serverUrl.doOnTextChanged { text, _, _, _ ->
+            if (text.isNullOrEmpty()) return@doOnTextChanged
+            when {
+                text.startsWith("http") -> {
```

Comment:
Theoretically someone could use `http` as an SSH user name. Let's treat `http://` and `https://` as the two possible prefixes for HTTP and show the SSH auth modes otherwise, no need for a third case.